packages/browserify/test/config.js

```diff
@@ -38,12 +43,14 @@ test('policy - writes a proper policy to a temp dir', async (t) => {
     },
     opts: {
       writeAutoPolicy: true
-    }
+    },
+    expectedResult: 555
   })
   const { projectDir } = await prepareScenarioOnDisk({ scenario, policyName: 'browserify' })
-  await runBrowserify({ projectDir, scenario})
-  const testResult = await runScenario({ scenario, dir: projectDir })
-  t.deepEqual(testResult, 555)
+  scenario.dir = projectDir
+  await runBrowserify({ scenario })
```

Comment:
what does runBrowserify do? vs runScenario? vs prepareScenarioOnDisk?

seems like `runScenario` should include prepareScenarioOnDisk and runAndTestScenario

Reply:
This test is redundant, the one above does essentially the same thing

Reply:
The reason why it was set like this is create the working temp dir, pass it to `runBrowserify` to run with writeAutoPolicy (which doesn't eval the bundle) in order to generate the policy. Then to run the scenario, this time evaling the bundle and producing the expected result.

Even this however, doesn't fully make sense because writeAutoPolicy isn't being turned off when running `runAndTestScenario`

I just removed it